### ISSUE TITLE
ConvertTo-PSFHashtable -Remap does not add other properties

### ISSUE THREAD

**Issue by danubie** [PowershellFrameworkCollective/psframework#587]
Hi,

I'm not sure if the old behaviour was unexpected but with version 1.8.289 ConvertTo-PSFHashtable changed its behaviour for handling properties not mentioned in -Include/Exclude list:

```powershell
Import-Module PSFramework -MaximumVersion 1.8.0
$obj = [pscustomobject] @{ prop1 = 1; prop2 = 2; prop3 = 3 }
ConvertTo-PSFHashtable -InputObject $obj -Exclude prop3 -Remap @{ prop2 = 'other2' }

Name                           Value
----                           -----
other2                         2
prop1                          1
```

```powershell
Import-Module PSFramework -MaximumVersion  1.9.0
$obj = [pscustomobject] @{ prop1 = 1; prop2 = 2; prop3 = 3 }
ConvertTo-PSFHashtable -InputObject $obj -Exclude prop3 -Remap @{ prop2 = 'other2' }

Name                           Value
----                           -----
other2                         2
```
The new version does not include all properties by default anymore. Adding them using -Include prop1 returns the same result as in version 1.7.270.

**Comment by danubie** [comment]
Thanks for the quick response. 
I recognized the nice idea to include the mapped properties automatically.
For me it's fine for now. I pinned the previous version in my psd1.

Thanks for you great work on this module. Could not be without it anymore 🥇

**Comment by FriedrichWeinmann** [comment]
Always great to be appreciated :)
But no need to pin, the next version is going to ship in an hour or two - already fixed, now writing the tests ... after which it's just a few pipelines until it is ready.

**Comment by FriedrichWeinmann** [comment]
Resolved and Released in v1.8.291
